app-misc/qmodmaster/qmodmaster-0.5.2.recipe

```diff
@@ -0,0 +1,87 @@
+SUMMARY="A free Qt-based implementation of a ModBus master application"
+DESCRIPTION="A graphical user interface allows easy communication with ModBus \
+ RTU and TCP slaves. QModMaster also includes a bus monitor for examining all  \
+ traffic in the bus
+ QModMaster is based on libmodbus. for modbus  \
+ communication and on QsLog for logging.
+ Supports Haiku, Windows and Linux.
+
+-Features
+     ModBus RTU/TCP Master
+     Monitor Bus activity [Raw Data and PDU details]
+"
+COPYRIGHT="2012-2020 Elbar"
+HOMEPAGE="https://sourceforge.net/projects/qmodmaster/"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://download.sourceforge.net/qmodmaster/qModMaster-code-0.5.2-3/qModMaster-code-$portVersion-3.zip
+CHECKSUM_SHA256="1754581e3059f14c31c584aa7abe1f1a602d1814f293823272b1158bbc179b81"
+SOURCE_FILENAME="qModMaster-code-$portVersion-3.zip"
+SOURCE_DIR="qModMaster-code-$portVersion"
+ADDITIONAL_FILES="qmodmaster.rdef.in"
```

Comment:
You forgot to add the rdef file in the PR :)

Reply:
@Begasus  thanks.. fixed